app/features/feedback/shared/strategies/drawing/column/feedback-mark.jsx

```diff
@@ -2,7 +2,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 function FeedbackMark({ rule }) {
-  const color = (rule.success) ? 'green' : 'red';
+  const color = (rule.success) ? (rule.falsePosMode ? 'red' : 'green') : 'red';
```

Comment:
So in false positive mode, both success and failure are red? Shouldn't failure be green?

Also, the linter should warn about nesting ternary operators like this. It's confusing to read and understand.

Declaring the colours separately might be easier to understand.
```js
const [ successColour, failureColour ] = rule.falsePosMode ? [ 'red', 'green' ] : [ 'green', 'red' ];
```

Reply:
> So in false positive mode, both success and failure are red? Shouldn't failure be green?
> 
No, not really, but only because your earlier statement:
> Ok, so it's a new class of rule that inverts the success/failure logic?

Is not correct. In false positive mode, the region doesn't render (the opacity gets set to zero), so the colour doesn't matter.

> Also, the linter should warn about nesting ternary operators like this. It's confusing to read and understand.
> 
> Declaring the colours separately might be easier to understand.
> 
> ```js
> const [ successColour, failureColour ] = rule.falsePosMode ? [ 'red', 'green' ] : [ 'green', 'red' ];
> ```

Will fix.